basics/variables.md

```diff
@@ -118,11 +118,40 @@ can also use `+=`, `-=`, `*=` and `/=` instead of `+`, `-`, `*` and
 `/`. The "advanced" `%=`, `//=` and `**=` also work.
 
 ```python
+>>> a = 11
 >>> a += 2          # a = a + 2
+>>> a
+13
+
+>>> a = 11
 >>> a -= 2          # a = a - 2
+>>> a
+9
+
+a = 11
 >>> a *= 2          # a = a * 2
+>>> a
+22
+
+>>> a = 11
 >>> a /= 2          # a = a / 2
->>>
+>>> a
+5.5
+
+>>> a = 11
+>>> a %= 2          # a = a % 2
+>>> a
+1
+
+>>> a = 11
+>>> a //= 2         # a = a // 2
+>>> a
+5
+
+a = 11
+>>> a **= 2         # a = a ^ 2
+>>> a
+121
 ```
```

Comment:
Do we really need this many examples? A couple examples would be good though, maybe with strings because not everyone like numbers and math.